add-ons/fabric8/fabric8.addon

```diff
@@ -0,0 +1,37 @@
+# Name: fabric8
+# Description: Installs fabric8 locally on Minishift
+# Required-Vars: GITHUB_OAUTH_CLIENT_ID,GITHUB_OAUTH_CLIENT_SECRET
```

Comment:
we need these so we can authenticate with github to be able to create github repos etc